src/node-sim.lisp

```diff
@@ -2,6 +2,10 @@
  
 (in-package :emotiq/sim)
 
+;; remove for production
+(defun eassert (bool-condition)
```

Comment:
@guitarvydas I really don't like this sort of interface.  

If you are going to cause the code to signal an error, you need to provide restarts for the user to choose (which is the whole point of `cl:assert` interface.

If you are simply trying to provide a test that the code does what one does, then use a test framework to write a unit test.

Writing a note to "Change for production" is the kind of thing that almost always gets overlooked when production actually happens.  

So, either factor these assertions out into a test, or provide a meaningful restart for the user.  

Since this is currently in the simulator which isn't necessarily going to be in production, I'm not going to neg. this pull request, but please figure out if this assertion should be correctable or be a unit test and change when you get a chance.